src/document/actor.js

```diff
@@ -159,15 +159,23 @@ export class WarhammerActor extends WarhammerDocumentMixin(Actor)
             for (let uuid of effectUuids)
             {
                 let effect = await fromUuid(uuid);
+                let message = game.messages.get(messageId);
```

Comment:
Why is this here? It's already declared above

Reply:
because i'm an idiot ;-). removed.